core/src/main/scala/filodb.core/metadata/Projection.scala

```diff
@@ -79,13 +79,13 @@ case class RichProjection(projection: Projection,
   def partitionKeyFunc: RowReader => PK =
     partitionType.getKeyFunc(partitionColIndices.toArray)
 
-  def toBinaryKeyRange[PK, SK](keyRange: KeyRange[PK, SK]): BinaryKeyRange =
+  def toBinaryKeyRange[P, S](keyRange: KeyRange[P, S]): BinaryKeyRange =
```

Comment:
Hmmm... guess I'm fine with making it single letter but then need to do it everywhere...